src/components/PriceBar/index.tsx

```diff
@@ -8,50 +8,37 @@ import TileColorBoard from "./TileColorBoard";
 
 const PriceBar: React.FC = () => {
   const [isOpen, setIsOpen] = React.useState<boolean>(true);
-  const drawerBtnStyles = useStyleConfig("DrawerButton", { isOpen });
-
-  const drawerController = () => setIsOpen(!isOpen);
+  const drawerBtnStyles = useStyleConfig("drawerButtonStyles", { isOpen });
 
   const drawerButton = (
-    controller: React.MouseEventHandler<HTMLDivElement | HTMLButtonElement>
-  ) => {
-    return (
-      <Box onClick={controller} sx={drawerBtnStyles} as="button" data-testid="drawer-button">
-        <Box pos="absolute" zIndex="1" top="50%" left="50%" transform="translate(-50%, -50%)">
-          {isOpen ? <Down /> : <Up />}
-        </Box>
+    <Box
+      as="button"
+      onClick={() => {
+        setIsOpen(!isOpen);
+      }}
+      sx={drawerBtnStyles}
+      data-testid="drawer-button"
+    >
+      <Box pos="absolute" zIndex="1" top="50%" left="50%" transform="translate(-50%, -50%)">
+        {isOpen ? <Down /> : <Up />}
       </Box>
-    );
-  };
+    </Box>
+  );
 
   return (
-    <Box position="fixed" bottom="0px" width="100vw" marginBottom="38px" zIndex="1600">
-      {!isOpen && drawerButton(drawerController)}
-      <Drawer
-        placement="bottom"
-        onClose={drawerController}
-        isOpen={isOpen}
-        closeOnOverlayClick={false}
-        variant="alwaysOpen"
-      >
-        <DrawerContent
-          backgroundColor="#fff"
-          sx={{
-            width: "calc(100% - 98px)",
-            marginLeft: "98px",
-            marginBottom: "42px",
-          }}
-        >
-          {drawerButton(drawerController)}
-          <DrawerBody boxShadow="0px -5px 10px -5px lightgrey" p="0">
+    <Box position="fixed" bottom="42px" right="0px " width="calc(100% - 98px)" zIndex="1600">
```

Comment:
Please remove the empty space in the right props.

Reply:
Thanks Tao zong, fixed :)